src/auth/authenticationManager.ts

```diff
@@ -51,3 +43,59 @@ export const useCheckMe = (path: string) => {
 
   return username;
 };
+
+export const useCheckIsLoggedIn = () => {
+  const isRefreshToken = myLocalStorage.isRefreshToken();
+  const [trigger, result] = useLazyGetIdAccessTokenQuery();
+  const [triggerLogOut, resultLogOut] = useLazyLogOutQuery();
+  const navigate = useNavigate();
+  const { pathname } = useLocation();
+  const [isLoading, setIsLoading] = useState(false);
+  const [isSuccess, setIsSuccess] = useState(false);
+
+  useEffect(() => {
+    if (isRefreshToken) {
+      setIsLoading(true);
+      trigger({});
+      if (result.isSuccess) {
+        setIsLoading(false);
+        setIsSuccess(true);
+        const idToken = sessionStorage.getItem('IdToken');
+        const accessToken = sessionStorage.getItem('AccessToken');
+        if (!idToken) {
+          sessionStorage.setItem('idToken', result.currentData.IdToken);
+        }
+        if (!accessToken) {
+          sessionStorage.setItem('AccessToken', result.currentData.AccessToken);
+        }
+      } else if (result.isError) {
+        setIsLoading(false);
+        setIsSuccess(false);
```

Comment:
if we did not get fresh tokens, then we end up without tokens in the sessionStorage. What should the app do in this case?